data/drules_proto.txt

```diff
@@ -116013,9 +115979,9 @@ cont {
   element {
     scale: 19
     symbol {
-      name: "auction-m"
+      name: "shop-m"
```

Comment:
you see the shop-auction icon was changed from `auction` to a generic `shop` on z19, that's because it was declared z18, not z18-; see below